lib/tasks/git-init.js

```diff
@@ -53,8 +48,19 @@ class GitInitTask extends Task {
     return execa('git', ['add', '.']);
   }
 
-  _gitCommit(commitMessage) {
-    return execa('git', ['commit', '-m', commitMessage], { env: this.buildGitEnvironment() });
+  _gitCommit() {
+    const template = require('lodash.template');
+    let commitTemplate = fs.readFileSync(path.join(__dirname, '../utilities/COMMIT_MESSAGE.txt'));
+    let commitMessage = template(commitTemplate)(pkg);
+    let env = this.buildGitEnvironment();
+    return execa('git', ['commit', '-m', commitMessage], { env })
+      .catch(error => {
+        if (error.message.indexOf('git config --global user') > -1) {
```

Comment:
ahh, good point

So we should tweak the check to be something like:

```
if (error && error.message && error.message.indexOf('git config --global user') > -1)` {
```

Sound good?